addons/lunch/static/src/css/lunch.css

```diff
@@ -1,17 +1,6 @@
 @charset "utf-8";
-.openerp .oe_lunch .oe_lunch_alert textarea {
-  background-color: #ffc7c7;
-  padding: 10px;
-  height: 1em;
-  margin-bottom: 20px;
-}
-.openerp .oe_lunch button.oe_button_add {
-  position: relative;
-  top: -2px;
-  left: 3px;
-}
-.openerp .oe_lunch button.oe_button_plus {
-  margin: -2px;
+.openerp .oe_lunch .oe_lunch_alert {
```

Comment:
New convention : changing the `oe_` prefix with `o_`. Do it only for the class specific to the module lunch.